pom.xml

```diff
@@ -427,6 +428,17 @@ under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <java>
+            <palantirJavaFormat>
+              <version>2.81.0</version>
```

Comment:
why do we need this here? It should come from parent pom

Reply:
Agreed, but how/where do I update the parent pom?

Reply:
what is the reason of such change?

Reply:
In maven-parent `<version.palantirJavaFormat>2.80.0</version.palantirJavaFormat>` is defined so we have a consistent version across all projects.

Reply:
@olamy The version that is used by `maven-parent` is incompatible with JDK 25 which I was using locally. That said, I'll remove the change from the PR.
@Bukama Thanks